File: src/Web/CombatAnalysis.EnhancedWebApp/CombatAnalysis.EnhancedWebApp.Server/Models/CombatPlayerModel.cs
namespace CombatAnalysis.EnhancedWebApp.Server.Models;

public class CombatPlayerModel
{
    public int Id { get; set; }

    public double AverageItemLevel { get; set; }

    public int ResourcesRecovery { get; set; }

    public int DamageDone { get; set; }

    public int HealDone { get; set; }

    public int DamageTaken { get; set; }

    public PlayerModel Player { get; set; }

    public int CombatId { get; set; }
}

File: src/Web/CombatAnalysis.EnhancedWebApp/combatanalysis.enhancedwebapp.client/src/features/gameLogs/types/CombatPlayerModel.ts
import type { PlayerModel } from "./PlayerModel";

export type CombatPlayerModel = {
    id: number;
    averageItemLevel: number;
    resourcesRecovery: number;
    damageDone: number;
    healDone: number;
    damageTaken: number;
    player: PlayerModel;
    combatId: number;
}